### DISCUSSION TITLE
how to make this dashboard default for ha?

### DISCUSSION THREAD

**Discussion by Mareckil** [matt8707/ha-fusion#225] [Q&A]
As above, It works brilliantly for me as for totally green beginner I was able to create nice looking first dashboard. But I do not know how to set it as my default dashboard for ha. I am using supervised Home Assistant.

**Comment by abeksis** [comment]
Look at the answer here [link](https://github.com/matt8707/ha-fusion/issues/228#issuecomment-1902683503)
